package.json

```diff
@@ -42,6 +42,7 @@
     "url": "https://github.com/lirantal/is-website-vulnerable.git"
   },
   "dependencies": {
+    "@slimio/async-cli-spinner": "^0.5.1",
```

Comment:
can we switch out this dependency in favor of using [ora](https://www.npmjs.com/package/ora) directly?

Reply:
I used `ora` at first, but it caused the CLI to hang after `chromeLauncher.launch()` is executed.
I believe it had to do with using `await`.

Reply:
@ahmedkrmn are you sure? seems to work just fine to me like this:

![image](https://user-images.githubusercontent.com/316371/66399716-cba84d80-e9e8-11e9-9d51-7bfb71539f43.png)

Reply:
@lirantal  I tested that out, but it still cause the console to hang after calling `spinner1.stop()`.

Reply:
@ahmedkrmn could you run another check? it works fine for me and I wonder if the difference is Windows vs macOS although I doubt that. If straight-up as above this isn't working we can try using Ora's spinner.promise() and provide it in the two needed places, but I wish not to if we can avoid it.

Reply:
@lirantal
![demo](https://user-images.githubusercontent.com/20889958/66438079-e223c880-ea2b-11e9-9afe-5d3823eb9779.gif)

Reply:
I see. Is this with the above code snippet I attached?
Perhaps you can use the spinner.promise way of doing it instead of await? see here for referece: https://github.com/sindresorhus/ora#orapromiseaction-options

I'd really like to avoid using the original method you have in the PR.

Reply:
@lirantal yeah this behavior is encountered when using your attached snippet.
What do you mean by:

> the original method you have in the PR